quill-sql/src/main/scala/io/getquill/context/sql/norm/RemoveReturning.scala

```diff
@@ -0,0 +1,15 @@
+package io.getquill.context.sql.norm
+
+import io.getquill.ast._
+
+object RemoveReturning extends StatelessTransformer {
```

Comment:
Considering that it also filters out the returning property, maybe this could be `NormalizeReturning`?